internal/config/config.go

```diff
@@ -194,7 +194,7 @@ func WithDefaults() *Config {
 		DriverName:                "godatabrickssqlconnector", // important. Do not change
 		ThriftProtocol:            "binary",
 		ThriftTransport:           "http",
-		ThriftProtocolVersion:     cli_service.TProtocolVersion_SPARK_CLI_SERVICE_PROTOCOL_V6,
+		ThriftProtocolVersion:     cli_service.TProtocolVersion_SPARK_CLI_SERVICE_PROTOCOL_V7,
```

Comment:
I think I was able to get NamedParameters working against e2-dogfood SQL without updating the ThriftProtocolVersion. Wondering if we know what changing this would achieve?